internal/models/users/usernames/usernames.go

```diff
@@ -0,0 +1,93 @@
+package usernames
+
+import (
+	"database/sql"
+	"math"
+)
+
+type UsernameAdminInfo struct {
+	Adjectives []string
+	Nouns      []string
+	APage      int
+	NPage      int
+	AWordCount int
+	NWordCount int
+}
+
+// UsernamesPerPage is the number of usernames to display per page
+var UsernamesPerPage = 25
```

Comment:
kinda weird having this as a global var, mby just include this from the endpoint so it can be modified at run time with query params?

Reply:
done, is now a query param